setup.py

```diff
@@ -35,11 +35,18 @@
         ]
     },
     install_requires=[
+        'pillow',
         'pandas',
         'pymongo',
         'pyserial',
         'six',
-        'utm'
+        'scipy',
+        'utm',
+        'matplotlib',
+        'jupyter',
```

Comment:
I don't think jupyter is a strict _requirement_ here, since it is only used for the notebooks.  It might make sense to make this an `extras_require` (http://peak.telecommunity.com/DevCenter/setuptools#declaring-extras-optional-features-with-their-own-dependencies) or omit it entirely here and put it into the `README.md` for the notebooks.  Having this here will make it hard to use this package for headless data packaging.